src/Plivo/Resources/Call/CallInterface.php

```diff
@@ -69,16 +69,16 @@ function __construct(BaseClient $plivoClient, $authId)
      * @return JSON output
      * @throws PlivoValidationException,PlivoResponseException
      */
-    public function create($from, array $to, $answerUrl, $answerMethod,
+    public function create($from, array $to, $answerUrl,
                            array $optionalArgs = [])
     {
         $mandatoryArgs = [
             'from' => $from,
             'to' => implode('<', $to),
-            'answer_url' => $answerUrl,
-            'answer_method' => $answerMethod
+            'answer_url' => $answerUrl
         ];
-
+        $d = implode('<',$to);
```

Comment:
I am using implode first since we are mentioning all the to numbers as a single element in an array. 
Using implode function I am converting it to a string first and later that string is converted to an array which will enable us to check all the values mentioned.